src/Layout/README.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Layout'
+type: 'component'
+components:
+- Layout
+categories:
+- Layout
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+---
+
+A wrapper component that allows to control the size of child blocks on different screen sizes.
+
+## Basic usage
+
+```jsx live
+  <Layout
+    lg={{ span: [4, 4, 4], offset: [0, 0, 0] }}
+    md={{ span: ['auto', 'auto', 'auto'], offset: [0, 0, 0] }}
+    sm={{ span: [8, 4, 6], offset: [0, 0, 6] }}
+    xs={{ span: [4, 4, 4], offset: [0, 0, 0] }}
+    xl={{ span: [3, 6, 3] }}
```

Comment:
Based on PWG meeting, let's move the API to support something like:

```jsx
<Layout
    lg={[{ span: 4, offset: 0}, { span: 4, offset: 0 }, { span: 4: offset: 0 }]}
/>
```

to make it more clear how the `span`/`offset` relate to each other per column. It's also a bit more in line with the `react-bootstrap` API on the `Col` component.